fastplotlib/graphics/selectors/_linear_region.py

```diff
@@ -398,13 +411,13 @@ def get_selected_indices(self, graphic: Graphic = None) -> Union[np.ndarray, Lis
         source = self._get_source(graphic)
 
         # if the graphic position is not at (0, 0) then the bounds must be offset
-        offset = getattr(source, f"position_{self.bounds.axis}")
-        offset_bounds = tuple(v - offset for v in self.bounds())
+        offset = getattr(source, f"position_{self._selection.axis}")
+        offset_bounds = tuple(v - offset for v in self._selection())
```

Comment:
once `_selection` is defined in the constructor just access it using the property, change the rest here too